### PR TITLE
Fix/heart position size wish list

### PR DESCRIPTION
**Pull request by ThaynanAndrey**
#### What is the purpose of this pull request?

Fix heart's position and size, and align products menu's button.

#### What problem is this solving?

The heart's position and size doesn't according expected template. Besides that, the products menu's button isn't aligned with products list.

#### How should this be manually tested?

1. Acess this [workspace](https://thaynan--storecomponents.myvtex.com/);
2. Selects a listed product on home page that contains more one image;
3. Observes that the heart's position isn't over image, but over small images;
Besides, observes that heart size is 16x16 px, when in fact it should be 32x32.

#### Types of changes

* [X] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by klzns — CHANGES_REQUESTED** [review]
Do not remove yarn.lock

**Comment by GustavoMA** [comment]
Your yarn deleted all `integrity sha512` checks. Check out your yarn version.

File: react/ProductDetails.js [hunk 1]
```diff
@@ -354,6 +357,11 @@ class ProductDetails extends Component {
     const showProductPrice =
       Number.isNaN(+availableQuantity) || availableQuantity > 0
 
+    
+    const classAddonDetails = classNames('absolute z-3 left-0', {
```
Comment (lucis) [inline react/ProductDetails.js]:
```suggestion
    const addonDetailsClasses = classNames('absolute z-3 left-0', {
```

**Review by lucis — APPROVED** [review]
Check if everything is fine with the `yarn.lock`, probably is!

Nice job ;)

**Review by klzns — APPROVED** [review]

**Review by oldufre — CHANGES_REQUESTED** [review]
O alinhamento e a distância tá errado, tá mais para cima do que para baixo Tem um padding de 4 e 12 a mais. Acho que isso que ta deixando a distancia errada.
São 2rem de distância da esquerda para direita e 2rem de distancia de cima para baixo.

image

**Review by vitoria — APPROVED** [review]
Very good! :clap:

**Review by oldufre — APPROVED** [review]

**Review by GustavoMA — APPROVED** [review]

**Pull request merged**